From d438d9ccebeca6bd710f6606985095cf4875d1e7 Mon Sep 17 00:00:00 2001
From: Zhaoyifeng <zyf@rock-chips.com>
Date: Tue, 21 Apr 2015 13:03:24 +0800
Subject: [PATCH]    NAND : update dts config for nand.    nandc0reg are unused
 now,rknand_base.c will check boot media,if boot form emmc,nand porbe will not
 enable clock.

---
 arch/arm64/boot/dts/rk3368-tb_8846.dts | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rk3368-tb_8846.dts b/arch/arm64/boot/dts/rk3368-tb_8846.dts
index 8f3eb6fd52c4..2a5ce0e89c0c 100755
--- a/arch/arm64/boot/dts/rk3368-tb_8846.dts
+++ b/arch/arm64/boot/dts/rk3368-tb_8846.dts
@@ -250,11 +250,13 @@
 };
 
 &nandc0 {
-	status = "disabled"; // used nand set "disabled" ,used emmc set "okay"
+/*set "okay" both nand and emmc*/
+/*if using nand,emmc need disabled*/
+	status = "okay";
 };
 
 &nandc0reg {
-	status = "okay"; // used nand set "disabled" ,used emmc set "okay"
+	status = "disabled"; /* unused now,set "disabled"*/
 };
 
 &emmc {
-- 
2.35.3

